Add more hyperparams for EvolutionOptimizer

UsingtcNower opened the pull request:
### Changes

<!--- What was changed (briefly), how to reproduce (if applicable), what the reviewers should focus on -->

### Reason for changes

<!--- Why should the change be applied -->

### Related tickets

<!--- Post the numerical ID of the ticket, if available -->

### Tests

<!--- How was the correctness of changes tested and whether new tests were added -->

ljaljushkin reviewed (changes requested):
Thanks for the contribution! Please address one comment.

UsingtcNower commented on nncf/config/schemata/defaults.py:
```diff
@@ -59,6 +59,10 @@
 PRUNING_LEGR_TRAIN_STEPS = 200
 PRUNING_LEGR_MAX_PRUNING = 0.8
 PRUNING_LEGR_RANDOM_SEED = 42
+PRUNING_LEGR_POPULATION_SIZE = 64
```
Good catch~ And thanks for your review! I've add it.

UsingtcNower commented:
> Thanks for the contribution! Please address one comment.

Hi @ljaljushkin ,Thanks for your review! I've addressed the comment, Could you please help to check it?

ljaljushkin commented:
@UsingtcNower `pre-commit-linters` job is failing because of some issues with code style.
please fix it: https://github.com/openvinotoolkit/nncf/blob/develop/CONTRIBUTING.md#code-style

UsingtcNower commented:
> @UsingtcNower `pre-commit-linters` job is failing because of some issues with code style. please fix it: https://github.com/openvinotoolkit/nncf/blob/develop/CONTRIBUTING.md#code-style

Hi @ljaljushkin Thanks for your review! I've reformat the code, could you please help to check it? BTW, I found the check tool also pointed out a code outside this change, so I'm not sure whether I need submit it or not.

ljaljushkin commented:
> > @UsingtcNower `pre-commit-linters` job is failing because of some issues with code style. please fix it: https://github.com/openvinotoolkit/nncf/blob/develop/CONTRIBUTING.md#code-style
> 
> Hi @ljaljushkin Thanks for your review! I've reformat the code, could you please help to check it? BTW, I found the check tool also pointed out a code outside this change, so I'm not sure whether I need submit it or not.

Feel free to submit all diff after `make pre-commit` run. 
on our practice, it affects only modified files.
